std_msgs/Header header
string location
string[] ssid 
float32[] mean_rss
float32[] mean_dist
